src/components/Textarea/index.js

```diff
@@ -211,7 +211,7 @@ TextArea.propTypes = {
     name: PropTypes.string,
     isDisabled: PropTypes.bool,
     topLabel: PropTypes.node,
-    state: PropTypes.oneOf(['success', 'warning', 'error']),
+    state: PropTypes.oneOf(['success', 'warning', 'error', '']),
```

Comment:
Why empty string? Just don't pass the state variable at all

Reply:
👍